partiql-plan/src/main/kotlin/org/partiql/plan/v1/rel/RelExcludeStep.kt

```diff
@@ -0,0 +1,40 @@
+package org.partiql.plan.v1.rel
+
+/**
+ * TODO DOCUMENTATION
+ */
+public interface RelExcludeStep : Rel {
+
+    public fun getSteps(): List<RelExcludeStep>
```

Comment:
nit: could be clearer to use `getSubsteps`

```suggestion
    public fun getSubsteps(): List<RelExcludeStep>
```

Reply:
Ack